.github/workflows/format.yml

```diff
@@ -0,0 +1,44 @@
+name: 👔 Format
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  format:
+    if: github.repository == 'eslint-community/eslint-plugin-security'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+
+      - name: ⎔ Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: 📥 Install deps
+        run: npm install
+
+      - name: 👔 Format
+        run: npm run format
+
+      - name: 💪 Commit
```

Comment:
There may be some debate as to whether this workflow should be introduced, but at least I think the code should be tested to ensure it's not broken. I think it's a good idea to test with the oldest Node version we support, as autofix may have replaced it with newer syntax.